python/pyspark/sql/readwriter.py

```diff
@@ -575,7 +575,7 @@ def bucketBy(self, numBuckets, col, *cols):
         .. note:: Applicable for file-based data sources in combination with
                   :py:meth:`DataFrameWriter.saveAsTable`.
 
-        >>> (df.write.format('parquet')
+        >>> (df.write.format('parquet')  # doctest: +SKIP
```

Comment:
Glad to know it. : ) For multi-line commands, we just need to do it in the first line?